<h1> # Original File is converted into CSV. </h1>


library(readr)
analysis.anova <- read.csv("C:/Users/radha/Desktop/KaranExcel/Analyst Case Study AutoFinance.csv", sep=",", header = TRUE)
head(analysis.anova)
str(analysis.anova)

<h1> Filter relevant columns 2,3,13. 2nd = city, 3rd = state, 13 = Monthly Income </h1>

In [4]:
col2 = 2
col3 = 3
col13 = 13
modifiedData = analysis.anova[,c(col2,col3,col13)]
write.csv(modifiedData,"C:\\Users\\radha\\Documents\\ClenaDataForTask1.csv")


In [10]:
# Load  new clean file for analysis
analysisAfterClean <- read.csv("C:\\Users\\radha\\Documents\\ClenaDataForTask1.csv", sep=",", header = TRUE)
head(analysisAfterClean)

X city               state monthlyIncome
1 1 NEW ALBANY         IN    2000         
2 2 BRONX              NY    2500         
3 3 HUNTINGTON STATION NY    4083         
4 4 Franklin           MI    4000         
5 5 BALTIMORE          MD    2200         
6 6 STAUNTON           IL    8750

<h2> Filter only ORANAGE, LOS ANGELS data </h2>

In [7]:
#Filter and only required data
Orange_LosAngels <- subset(analysisAfterClean, (city=="ORANGE" | city=="LOS ANGELES") & state=="CA") 

In [11]:
## count, monthly Income
library(dplyr)
result <- Orange_LosAngels %>% group_by(city) %>%  summarize(n(), sum(monthlyIncome) ,mean(monthlyIncome))

<h2> Mean Monthly Income of LOS ANGELS and ORANGE </h2>

In [9]:
print(result)

# A tibble: 2 x 4
  city        `n()` `sum(monthlyIncome)` `mean(monthlyIncome)`
  <fct>       <int>                <int>                 <dbl>
1 LOS ANGELES  1018              3164913                 3109.
2 ORANGE         56               290353                 5185.


<h3>In Monthly Income - Mean - USD</h3>
<h3> LOS ANGELS - 3109 </h3>
<h3> ORANGE -  5185</h3>